content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -197,6 +231,7 @@ data:
     .:53 {
         errors
         health
+        ready
```

Comment:
fwiw, I have to use this to get things moving:

```
Stevens-MacBook-Pro-2:multicloud-sdake sdake$ cat coredns.sh
kubectl apply -f - <<EOF
apiVersion: v1
kind: ConfigMap
metadata:
  name: coredns
  namespace: kube-system
data:
  Corefile: |
    .:53 {
        errors
        ready
        health
        kubernetes cluster.local in-addr.arpa ip6.arpa {
           pods insecure
           upstream
           fallthrough in-addr.arpa ip6.arpa
        }
        prometheus :9153
        forward . /etc/resolv.conf
        cache 30
        loop
        reload
        loadbalance
    }
    global:53 {
        errors
        ready localhost:9196
        health localhost:9197
        cache 30
        forward . $(kubectl get svc -n istio-system istiocoredns -o jsonpath={.spec.clusterIP}):53
    }
EOF
```

Reply:
changes include using `:53` at the conclusion of the forward line and putting ready and health on their own ports as coredns complains about port in use problems.

Reply:
What CoreDNS version are you using ? I suspect that there are some slight differences between them.

Reply:
I added the port (which makes definitely sense) but the ready and health statements can be dropped since we don't check them anyway?

Reply:
wfm.